### ISSUE TITLE
Issue on page /intermediate/hvplot.html

### ISSUE THREAD

**Issue by lealiaxiong** [xarray-contrib/xarray-tutorial#304]
The interactivity examples for using hvplot with xarray are not working for me. I am following the tutorial, with the example data. While the interactive plots seem to render and display, the slider does not cause the plot to change. I am attaching a screen recording. 

https://github.com/user-attachments/assets/94980e26-690f-45dd-9ad3-4ed8d6d14243

hvplot version: 0.7.3
bokeh version: 3.6.0
holoviews version: 1.20.0

**Comment by scottyhq** [comment]
Yeah, it's a known issue, workarounds described here https://github.com/xarray-contrib/xarray-tutorial/pull/268
